package/yast2-services-manager.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Wed Jan 13 09:00:17 UTC 2016 - igonzalezsosa@suse.com
+
+- Add X-SuSE-YaST-AutoInstSchema declaration so
+  services-manager.rnc could be included in yast2-schema package
+- Fix validation of AutoYaST profile (bsc#954412)
+- 3.1.40.1
```

Comment:
Mmm, I think it would be useful to backport the fix in 3.1.41 to the SP1 branch just in order to not diverge. It looks absolutely safe and diverging just because of that it's a shame.